sdk/formrecognizer/ai-form-recognizer/src/models.ts

```diff
@@ -954,3 +956,47 @@ export interface AccountProperties {
    */
   customModelLimit: number;
 }
+
+/**
+ * Request parameter that contains authorization claims for copy operation.
+ */
+export type CopyAuthorization = {
+  /**
+   * Target resource Id.
+   */
+  resourceId: string;
+  /**
+   * Target resource region.
+   */
+  resourceRegion: string;
+  /**
+   * The time when the access token expires.
+   */
+  //expiresOn: Date
+} & CopyAuthorizationResultModel;
+
+/**
+ * Status and result of the queued copy operation.
+ */
+export interface CopyResult {
```

Comment:
is this an internal type?

Reply:
This was the result from copy function, but no longer used, I will remove.

Reply:
Removed in https://github.com/Azure/azure-sdk-for-js/pull/9068/commits/646b9d0df146a3129b8d1685a377e69200aeb913